MerlinAU.sh

```diff
@@ -6204,7 +6210,7 @@ do
           else _Approve_FW_Update_
           fi
           ;;
-      up) _SCRIPTUPDATE_
+      up) _SCRIPTUPDATE_ "$forceScriptUpdateOption"
```

Comment:
Ah, I see the bug. 
I forgot to include the optional "force" for this.

Reply:
Actually, this is not the exact design I intended.
I'll submit another PR with the suggested changes of the original design choices I intended to do.

Let me know what you prefer.

Reply:
So my intention was not to use the force here actually.
It was to use the regular update function. 

The bug, was actually in my _SCRIPTUPDATE_ function when setting: `if [ "$1" = "force" ]`